app/managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table} (first_name, last_name) "
+            f"VALUES (?, ?) ",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        cursor = self._connection.execute(
+            f"SELECT * FROM {self.table}"
+        )
+        return [
+            Actor(*row) for row in cursor
+        ]
+
+    def update(
+            self,
+            id_to_update: str,
```

Comment:
same here , why it is called id_to_update if it actually an actor id ?)

Reply:
we had name like this in the video, so i thought this is a good name 
![image](https://github.com/mate-academy/py-actor-manager/assets/105279117/0e7efa58-811d-4056-85ee-0ca52337f937)

Reply:
Imho that was more for example : )